.github/workflows/build_packages.yaml

```diff
@@ -224,9 +229,10 @@ jobs:
         path: source/_packages/${{ matrix.profile }}/.
 
   linux:
-    runs-on: ubuntu-20.04
-
     needs: prepare
+    runs-on: ${{ matrix.build_machine }}
+    container:
+      image: "ghcr.io/emqx/emqx-builder/5.0-3:${{ matrix.otp }}-${{ matrix.os }}"
```

Comment:
maybe keep building from `scripts/buildx.sh`
it as no performance loss when it's not a 'cross' build

Reply:
Like in L25, we don't want to install erlang/rebar3 and another toolchain, it is always some command unavailable on the runner to make the job run.

Reply:
Since I have moved the task id: 'deps-refs' into job 'prepare, we might not need rebar3 in this job. 

I could try to revert the change but I remember I had a hard time with permissions (docker run change permission to root) since I don't want self-hosted runner users to have sudo access.

Reply:
No, No need to revert.
Your argument about rebar3 makes a lot of sense. it's better to do everything in the container.